backend/src/test/java/ai/verta/modeldb/ExperimentRunTest.java

```diff
@@ -8489,29 +8490,36 @@ public void logGetVersioningExperimentRunCreateTest()
             .build();
     experimentRunServiceStub.logVersionedInput(logVersionedInput);
 
-    logVersionedInput =
-        LogVersionedInput.newBuilder()
-            .setId(experimentRun.getId())
-            .setVersionedInputs(
-                VersioningEntry.newBuilder()
-                    .setRepositoryId(repoId)
-                    .setCommit(commitResponse.getCommit().getParentShasList().get(0))
-                    .build())
-            .build();
-    experimentRunServiceStub.logVersionedInput(logVersionedInput);
+    LogVersionedInput logVersionedInputFail = LogVersionedInput.newBuilder()
+        .setId(experimentRun.getId())
+        .setVersionedInputs(
+            VersioningEntry.newBuilder()
+                .setRepositoryId(repoId)
+                .setCommit(commitResponse.getCommit().getParentShasList().get(0))
+                .build())
+        .build();
+    try {
+      experimentRunServiceStub.logVersionedInput(logVersionedInputFail);
+      fail();
+    } catch (StatusRuntimeException e) {
+      Status status = Status.fromThrowable(e);
+      LOGGER.warn("Error Code : " + status.getCode() + " Description : " + status.getDescription());
+      assertEquals(Status.ALREADY_EXISTS.getCode(), status.getCode());
+    }
+
+    try {
+      experimentRunServiceStub.logVersionedInput(logVersionedInputFail);
+      fail();
+    } catch (StatusRuntimeException e) {
+      Status status = Status.fromThrowable(e);
+      LOGGER.warn("Error Code : " + status.getCode() + " Description : " + status.getDescription());
+      assertEquals(Status.ALREADY_EXISTS.getCode(), status.getCode());
+    }
```

Comment:
I think this is redundant. this should be a single case.

Reply:
removed it